www_src/pages/make/make.jsx

```diff
@@ -1,32 +1,56 @@
 var React = require('react');
 var render = require('../../lib/render.jsx');
-var Binding = require('../../lib/binding.jsx');
 var api = require('../../lib/api.js');
 var Card = require('../../components/card/card.jsx');
 var Link = require('../../components/link/link.jsx');
 
 var Make = React.createClass({
-  mixins: [Binding],
+  mixins: [],
+  getInitialState: function () {
+    return {
+      projects: []
+    };
+  },
   componentWillMount: function () {
+    this.load();
+  },
+  componentDidUpdate: function (prevProps) {
+    if (this.props.isVisible && !prevProps.isVisible) {
+      this.load();
+      console.log('restored!');
+    }
+  },
+  load: function () {
     api({
-      uri: '/c0645e6953e9949f8e5c/raw/'
-    }, function (err, body) {
-      console.dir(err);
-      console.dir(body);
+      uri: '/users/1/projects'
+    }, (err, body) => {
+      if (err) return console.error('Error getting projects', err);
```

Comment:
I wonder if we should do the `err` logging within the API module itself as to reduce this boilerplate and instead have a clean base for handling errors in the follow-up work. Perhaps even move the logs into the API module and replace all of these with `@todo`.